scenarios/kubernetes_e2e.py

```diff
@@ -473,6 +479,9 @@ def create_parser():
     parser.add_argument(
         '--kubeadm', choices=['ci', 'periodic', 'pull'])
     parser.add_argument(
+        '--kubeadm-kubernetes-version', default=None, help='Version of Kubernetes to use while '
+        'testing kubeadm (e.g. latest, latest-1.6, 1.6.4, etc)')
```

Comment:
you can just pass in `--kubernetes-anywhere-kubernetes-version=x.x.x` from config.json, rest should be automatic